store/src/java/com/zimbra/util/ExternalVolumeInfoHandler.java

```diff
@@ -157,13 +163,28 @@ public void addServerProperties(VolumeInfo volInfo, String serverId) throws Serv
                     && AdminConstants.A_VOLUME_S3.equalsIgnoreCase(volInfo.getVolumeExternalInfo().getStorageType())
                     && volInfo.getVolumeExternalInfo().isUnified()) {
                 editGlobalConfigOnAddVolume(volInfo, serverId);
+                flushCache(zsc);
             }
-
         } catch (JSONException e) {
             throw e;
         }
     }
 
+    /**
+     * Flushes config level cache of all other server
+     * @param zsc
+     * @return nothing
+     */
+    public void flushCache(ZimbraSoapContext zsc) {
```

Comment:
I would name it to specific attributes

Reply:
Renamed method name to flushConfigLevelCacheOnAllServers().